install_config/persistent_storage/persistent_storage_glusterfs.adoc

```diff
@@ -13,567 +14,208 @@ toc::[]
-----
-<1> `*securityContext*` must be defined at the pod level, not under a specific container.
-<2> An array of GIDs defined at the pod level.
-
-Assuming there are no custom SCCs that satisfy the pod's requirements, the pod
-matches the *restricted* SCC. This SCC has the `*supplementalGroups*` strategy
-set to *RunAsAny*, meaning that any supplied group IDs are accepted without
-range checking.
-
-As a result, the above pod will pass admissions and can be launched. However, if
-group ID range checking is desired, use a custom SCC, as described in
-xref:pod_security_context.adoc#scc-supplemental-groups[pod security and custom
-SCCs]. A custom SCC can be created to define minimum and maximum group IDs,
-enforce group ID range checking, and allow a group ID of *590*.
-
-[[gfs-user-ids]]
-=== User IDs
-
-User IDs can be defined in the container image or in the pod definition. The
-full  xref:pod_security_context.adoc#user-id[Volume Security] topic covers
-controlling storage access based on user IDs, and should be read prior to
-setting up NFS persistent storage.
-
-[NOTE]
-====
-Use xref:gfs-supplemental-groups[supplemental group IDs] instead of user IDs to
-gain access to persistent storage.
-====
-
-In the xref:gfs-mount[target Gluster mount example above], the container needs
-a UID set to *592*, so the following can be added to the pod definition:
-
-[source,yaml]
-----
-spec:
-  containers: <1>
-  - name:
-  ...
-    securityContext:
-      runAsUser: 592 <2>
-----
-<1> Pods contain a `*securtityContext*` specific to each container and a pod-level `*securityContext*`, which applies to all containers defined in the pod.
-<2> The UID defined on the Gluster mount.
-
-With the *default* project and the *restricted* SCC, a pod's requested user ID
-of *592* will not be allowed, and the pod will fail. This is because:
-
-- The pod requests *592* as its user ID.
-- All SCCs available to the pod are examined to see which SCC will allow a user ID of *592*.
-- Because all available SCCs use *MustRunAsRange* for their `*runAsUser*`
-strategy, UID range checking is required.
-- *592* is not included in the SCC or project's user ID range.
-
-Do not modify the predefined SCCs. Insead,
-xref:pod_security_context.adoc#scc-runasuser[create a custom SCC] so that
-minimum and maximum user IDs are defined, UID range checking is still enforced,
-and the UID of *592* will be allowed.
-
-[[selinux]]
-=== SELinux
-
-[NOTE]
-====
-See the full xref:../../install_config/persistent_storage/pod_security_context.adoc#selinuxoptions[Volume Security] topic for
-information on controlling storage access in conjunction with using SELinux.
-====
+[[provisioning]]
+== Provisioning
 
-By default, SELinux does not allow writing from a pod to a remote Gluster
-server.
+GlusterFS volumes can be provisioned either statically or dynamically. Static
+provisioning is available in all configurations. Only {gluster-native} and
+{gluster-external} support dynamic provisioning.
```

Comment:
This should be re-worded as `{gluster-native}` and `{gluster-external}` are the only options we explain to customers.

Reply:
We have several customers and upstream users using standalone RHGS/GlusterFS with static provisioning for OCP. I think the current wording is accurate and clear. I'm open to suggestions, however.